lib/active_fedora/aggregation/base_extension.rb

```diff
@@ -3,6 +3,16 @@ module BaseExtension
     extend ActiveSupport::Concern
     include PersistLinks
 
+    # Queries the RDF graph to find all aggregations that include this object
+    def aggregated_by
+      # In theory you should be able to find the aggregation predicate (ie ore:aggregates)
+      # but Fedora does not return that predicate due to this bug in FCREPO: https://jira.duraspace.org/browse/FCREPO-1497 
+      # so we have to, instead, look up the proxies asserting RDF::Vocab::ORE.proxyFor and return their containers.
+      proxy_uris = self.resource.query(predicate: RDF::Vocab::ORE.proxyFor, object: RDF::URI(self.uri)).subjects.to_a
```

Comment:
I don't understand what you are suggesting.  If you want to offer a replacement for this line, I will try it.